test/team_stats_test.rb

```diff
@@ -6,12 +6,18 @@
 class TeamStatsTest < Minitest::Test
 
   def setup
-    @team_stats = TeamStats.new("./data/teams.csv", TeamStats)
```

Comment:
Another good catch

Reply:
Thanks for finding that - I introduced this bad line to the code (sorry),.